CHANGELOG.md

```diff
@@ -1,3 +0,0 @@
-## 1.0.0 / unreleased
-
-Initial release of the driver
```

Comment:
Why you've deleted this file?

Reply:
We should just use git(hub)'s release mechanism (which is necessary for packagist anyway), and not duplicate the funcitonality.

This is to avoid questions later like:
- is the necessary
- what's the format
- what if it's forgotten
- mentioning it in other places (like contributing.md)

Reply:
OK. Let's see how it works out in practice. We can always add that file later on.